src/webui/www/private/scripts/misc.js

```diff
@@ -120,3 +120,9 @@ function escapeHtml(str) {
     div.appendChild(document.createTextNode(str));
     return div.innerHTML;
 }
+
+function safeTrim(value) {
```

Comment:
>String(null) returns "null" and String(undefined) returns "undefined" in JavaScript.

OK, however I think a true "safe trim" should look like this:
```js
function safeTrim(value) {
    try {
        return value.trim();
    }
    catch (e) {
        if (e instanceof TypeError)
            return "";
        throw e;
    }
}
```

Reply:
Sure, will update the implementation.